server-common/src/main/java/org/apache/kafka/server/common/GroupVersion.java

```diff
@@ -22,7 +22,7 @@
 public enum GroupVersion implements FeatureVersion {
 
     // Version 1 enables the consumer rebalance protocol (KIP-848).
-    GV_1(1, MetadataVersion.IBP_4_0_IV0, Collections.emptyMap());
+    GV_1(1, MetadataVersion.IBP_3_9_IV0, Collections.emptyMap());
```

Comment:
Should we keep it attached to `IBP_4_0_IV0` as we already know that it will be release in 4.0? If not, we should file a blocker for 3.9 to ensure that we change it before we release 3.9.

Reply:
I personally would leave it attached to `IBP_4_0_IV0`. Even if there's a 3.10, we still know GV_1 is in 4.0.

Reply:
I can leave it attached to IBP_4_0_IV0.

Reply:
We need to revert this change.

Reply:
@dajac : Done